packages/arb-token-bridge-ui/src/constants.ts

```diff
@@ -3,23 +3,23 @@ export const TOS_VERSION = 1
 export const TOS_LOCALSTORAGE_KEY = 'arbitrum:bridge:tos-v' + TOS_VERSION
 
 export const GET_HELP_LINK =
-  'https://arbitrumfoundation.zendesk.com/hc/en-us/requests/new?ticket_form_id=18155929976987'
+  'https://support.arbitrum.io/hc/en-us/requests/new?ticket_form_id=18155929976987'
```

Comment:
Can we use a variable for `https://support.arbitrum.io` part? So we don't copy paste it everywhere?

Reply:
updated